### PR TITLE
[as7816-64x]Modify to check specific DUT

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

#### Why I did it
AS7816 support  AT or non-AT DUT. They use different pmbus i2c bus. So use "pre_pddf_init.sh" to check this case.
#### How I did it
Add pre_pddf_init.sh.
#### How to verify it
Add pre_pddf_init.sh to build image. Load to check on AT DUT. PSU work well.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: device/accton/x86_64-accton_as7816_64x-r0/pddf/pddf-device.json [hunk 1]
```diff
@@ -3182,8 +3182,8 @@
 			"i2c" : {
 				"attr_list":
 				[
-					{"attr_name":"STATUS_LED_COLOR_AMBER", "descr" : "Amber", "bits" : "4",  "value" : "0x0", "swpld_addr" : "0x60", "swpld_addr_offset" : "0x30"},
-					{"attr_name":"STATUS_LED_COLOR_OFF",  "descr" : "Off", "bits" : "4",  "value" : "0x1", "swpld_addr" : "0x60", "swpld_addr_offset" : "0x30"}
+					{"attr_name":"amber", "descr" : "Amber", "bits" : "4",  "value" : "0x0", "swpld_addr" : "0x60", "swpld_addr_offset" : "0x30"},
```
Comment (FuzailBrcm) [inline device/accton/x86_64-accton_as7816_64x-r0/pddf/pddf-device.json:3185]: Please remove these changes

Comment (jostar-yang) [reply device/accton/x86_64-accton_as7816_64x-r0/pddf/pddf-device.json]: done to modify back pddf-device.json

**Review by FuzailBrcm — APPROVED** [review]
Changes look good to me

**Review by FuzailBrcm — APPROVED** [review]

**Pull request merged**